models/labels/dex/Persona/trader_platforms/labels_trader_platforms.sql

```diff
@@ -1,8 +1,48 @@
 {{
     config(
         alias='trader_platforms',
-        post_hook='{{ expose_spells(\'["ethereum"]\', "sector", "labels", \'["gentrexha"]\') }}'
+        post_hook='{{ expose_spells(\'["ethereum", "fantom", "arbitrum", "avalanche_c", "gnosis", "bnb", "optimism", "polygon"]\',
+                    "sector",
+                    "labels",
+                    \'["gentrexha", "Henrystats"]\') }}'
     )
 }}
 
-SELECT * FROM {{ ref('labels_trader_platforms_ethereum') }}
\ No newline at end of file
+with trader_platforms as (
+    select taker           as address,
+           MIN(block_time) as first_trade,
```

Comment:
It seems `first_trade` column is not used in the later select part.
We may remove the `order by first_trade`, though it might be a useful feature.

Reply:
I think it was intentionally used but will defer to @gentrexha on that.

Reply:
The `order by first_trade` keeps the order inside of the label on which platform they've used first.

Reply:
I see, so the platforms' name will be ranked in order.